### ISSUE TITLE
generators for different ecosystems/languages

### ISSUE THREAD

**Issue by jonschlinkert** [regexhq/regexhq#5]
cc/ @regexhq/owners 

Might help to get everyone on board with whatever convention we decided on by creating/publishing generators that use those conventions. 

Here are a couple of ideas just to get this started: 
- [ ] generator-regexhq (me or @sindresorhus - if he's interested - can do this one)
- [ ] generate-regexhq ([generate](https://github.com/generate) which is behind [verb](https://github.com/verbose/verb))
- [ ] component?
- [ ] khaos-regexhq (may not need publish, but the repo name)

Anyone can feel free to edit this issue to update the list

**Comment by tunnckoCore** [comment]
Only to point. Isn't it be better to be as organization name - generator-regexps? IMO, it will be confusing what generator exactly does if name is `regex(p)` (without the `s`)

**Comment by jonschlinkert** [comment]
I don't think so, we're not trying to create a "generator ecosystem". We just want to be able to make regex projects easily. I personally don't want to maintain generators as a goal per se.

**Comment by tunnckoCore** [comment]
@regexhq/owners, added `khaos`, `verb` and `generate` to the list

**Comment by tunnckoCore** [comment]
Now sounds better - RegexHQ - `generate-regexhq`, `generator-regexhq` :)

edit: List updated. Is there need for `component`? I don't think so.
